src/Concerns/WithCount.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Maatwebsite\Excel\Concerns;
+
+interface WithCount
```

Comment:
I'm not 100% satisfied with the naming. It doesn't really reveil what it is used for.  Don't have a better name immediately, but maybe you have some more suggestions?

Reply:
What about Countable?

Reply:
The word count might confuse people to think that they can retrieve a count from the export instead of the oppositie way. We need to reveil more that it's used for the chunk calculation.

Reply:
Perhaps `WithCustomQuerySize` ? Or even `WithCustomChunks` and giving control to both settings (count and chunk size)

Reply:
I would go with the first option then. Keeps it to one responsibility and, either way, users already have control of the chunk size through a configuration option, right?

Reply:
Can you make it `WithCustomQuerySize` then :)